ansible_collections/arista/avd/roles/eos_designs/defaults/main/default-node-type-keys.yml

```diff
@@ -25,3 +25,20 @@ default_node_type_keys:
     overlay_controller:
       type: overlay-controller
       default_evpn_role: server
+
+  "mpls-vpn":
+    p:
+      type: p
+      default_evpn_role: none
+    pe:
+      type: pe
+      connected_endpoints: true
+      default_evpn_role: client
+      network_services:
+        l1: true
+        l2: true
+        l3: true
+      vtep: true
```

Comment:
Is this applicable here?

Reply:
Deprecated use of vtep variable for mpls-vpn design, implemented mpls_tep feature toggle instead.